pkg/model/manifest.go

```diff
@@ -638,10 +647,37 @@ func Read(bytes []byte) (*Manifest, error) {
 	if err := manifest.setDefaults(); err != nil {
 		return nil, err
 	}
+
+	if err := manifest.validate(); err != nil {
+		return nil, err
+	}
 	manifest.Manifest = bytes
 	return manifest, nil
 }
 
+func (m *Manifest) validate() error {
```

Comment:
why don't we use `validateDivert` directly? this is for adding other validations in the future?

Reply:
yes, that's the idea. we could already move logic here, but I didn't do it on this PR